content/en/blog/_posts/2024-04-15-devsecops-in-action-k8s-secrets.md

```diff
@@ -0,0 +1,535 @@
+---
+layout: blog
+title: "DevSecOps in Action: Kubernetes Secrets"
+slug: devsecops-in-action-k8s-secrets
+date: 2024-04-15
+---
+
+**Author**: Evgenii Frikin
```

Comment:
If you rebase against _main_ you can (and should) use the `author` field in front matter. See other blog articles from 2024.

Reply:
Thank you for your suggest!

Fixed.